lms/envs/common.py

```diff
@@ -881,6 +881,15 @@
     #   toggle does not have a target removal date.
     'ENABLE_AUTHN_MICROFRONTEND': os.environ.get("EDXAPP_ENABLE_AUTHN_MFE", False),
 
+    # .. toggle_name: FEATURES['ENABLE_CATALOG_MICROFRONTEND']
```

Comment:
Just confirming, you will not be adding a Waffle flag (for per-user rollout), correct? This is OK, as long as you are OK with the options being global-on and global-off.

(If you were planning to add a Waffle flag too, then I would say just add the Waffle flag and get rid of this feature flag).